Servers/utils/eu.utils.ts

```
@@ -432,42 +449,45 @@ export const getCompliancesEUByIdQuery = async (
+    controlIds[0].map((c) => c.id), tenant
   );
   return compliances;
 };
 
 export const createNewAssessmentEUQuery = async (
   assessment: AssessmentEU,
   enable_ai_data_insertion: boolean,
+  tenant: string,
   transaction: Transaction,
   is_mock_data: boolean
 ): Promise<Object> => {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)